src/includes/breadcrumbs/automatic-breadcrumbs/flutter.mdx

```diff
@@ -0,0 +1,20 @@
+The Flutter SDK already captures breadcrumbs automatically via the Native SDKs.
+
+- [Automatic Breadcrumbs for Android](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+- [Automatic Breadcrumbs for iOS](/platforms/apple/usage/#sentryautobreadcrumbtrackingintegration)
+
+If you wish to add additional Navigation breadcrumbs for Flutter Apps, Add the `SentryNavigatorObserver` to your `MaterialApp`, `WidgetsApp` or `CupertinoApp`
```

Comment:
we should also mention here the `http` breadcrumbs available thru our the Dart SDK `platforms/dart/usage/advanced-usage/#automatic-breadcrumbs`